content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -33,8 +33,11 @@ function relativeTime(timestamp) {
   return new Date(timestamp).toLocaleString();
 }
 
+
+//https://spocs-staging.web.readitlater.com/v3/newtab/layout?version=1&consumer_key=$apiKey&layout_variant=staging_spocs
 const LAYOUT_VARIANTS = {
   basic: "Basic default layout (on by default in nightly)",
+  staging_spocs: "A layout with all spocs shown",
```

Comment:
This layout is hard to test as no spocs are getting returned from spoc_endpoint https://spocs.getpocket.com/spocs?site=1084367, I assume it should show spocs in all positions once we have spoc data

Reply:
Yeah in general the layout is still in flux, but I didn't want it to block review.